### PR TITLE
Globalize redis

### PR DESCRIPTION
**Pull request by jannyHou**
Connect to strongloop/loopback#2422

### CODE REVIEW

**Comment by jannyHou** [comment]
This is the globalization pr for loopback-connector-redis.
@0candy PTAL. Thanks!

File: lib/redis.js [hunk 1]
```diff
@@ -4,6 +4,9 @@
 var redis = require('redis');
 var async = require('async');
 
+var SG = require('strong-globalize');
+var g = SG();
```
Comment (0candy) [inline lib/redis.js]: Should move these two lines to the beginning of the file.

**Comment by jannyHou** [comment]
@superkhau @0candy feedback applied. PTAL
Thanks.

**Comment by 0candy** [comment]
LGTM :+1:

**Pull request merged**